shamir_mnemonic/__init__.py

```diff
@@ -515,44 +527,37 @@ def _generate_random_identifier():
     return identifier & ((1 << ID_LENGTH_BITS) - 1)
 
 
-def generate_mnemonics(
-    group_threshold, groups, master_secret, passphrase=b"", iteration_exponent=0
+def generate_mnemonics_from_data(
+    encrypted_master_secret, identifier, group_threshold, groups, iteration_exponent=0
```

Comment:
Just a note: we are currently using `iteration_exponent=1` in Core. I'm not saying we can't have the default to 0, but maybe it makes sense to remove the default value completely.

Reply:
Let's have `iteration_exponent=1` here as well.